### PR TITLE
Remove metrics from main branch 

### PR DESCRIPTION
**Pull request by lzchen**
Core pr: https://github.com/open-telemetry/opentelemetry-python/pull/1568

As part of the discussion [here](https://github.com/open-telemetry/opentelemetry-python/issues/1547#issuecomment-768592654)

### CODE REVIEW

**Review by ocelotl — COMMENTED** [review]
Disregard

**Review by ocelotl — APPROVED** [review]

**Review by codeboten — APPROVED** [review]
Just one question, otherwise it looks good.

File: instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py [hunk 1]
```diff
@@ -186,10 +165,7 @@ def process_response(self, request, response):
                 "{} {}".format(response.status_code, response.reason_phrase),
                 response,
             )
-            # pylint:disable=W0212
-            request._otel_labels["http.status_code"] = str(
-                response.status_code
-            )
```
[the remark this answers is not in the document]
Comment (lzchen) [reply instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py:192]: It was not removed accidently. `request.otel_labels` was added as a way to pass the labels needed to create duration metrics.

**Comment by mihirg** [comment]
is there an alternative available for this package?

**Comment by lzchen** [comment]
@mihirg 
If you mean where the metrics functionality is found, you can find it under the `metrics` branch in this repo. It will be released as experimental once our 1.0.0 packages goes GA.

**Comment by luckydonald** [comment]
The docs seem broken in the process?

**Comment by lzchen** [comment]
@luckydonald 
Which docs are you referring to?

**Pull request merged**